task/opm-render-bundles/0.1/opm-render-bundles.yaml

```diff
@@ -0,0 +1,72 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: opm-render-bundles
+spec:
+  description: Create a catalog index and render the provided bundles into it
+  params:
+    - name: binary-image
+      description: Image in which to build catalog
+      default: registry.redhat.io/openshift4/ose-operator-registry:latest
```

Comment:
Does the image need to be configurable? We've generally moved away from that because it makes it harder to "trust" the tasks. If the user can override the execution environment, we can't be sure what the task is going to do

Reply:
https://github.com/konflux-ci/build-definitions/pull/1170#discussion_r1693278155 oh I see, if it's just a configurable FROM line that should be fine